public/scripts/report.js

```diff
@@ -1,24 +1,80 @@
 (function(){
 
-  // GET LOCATION
-  var locationButton = document.getElementById('getLocationButton');
-
-  function getLocation() {
-    if (navigator.geolocation) {
-      navigator.geolocation.getCurrentPosition(showPosition);
-    } else {
-      // non-supprt error handling
-      console.log('Geolocation is not supported by this browser.');
+  // GET USER'S LOCATION
+  function getLocation(){
+
+    var locationButton = document.getElementById('getLocationButton');
+
+    function getLocation() {
```

Comment:
You've written two getLocation functions, so the second overwrites the first! Maybe create one big event listener function for adding both event listeners and call that at the start